CHANGELOG.md

```diff
@@ -41,6 +41,30 @@
 
   To learn more about the available configuration options visit [the documentation](https://docs.sentry.io/platforms/react-native/user-feedback/configuration).
 
+- Adds the ability to consider the `SENTRY_DISABLE_AUTO_UPLOAD` and `SENTRY_DISABLE_NATIVE_DEBUG_UPLOAD` environment variables in the Sentry Android Gradle Plugin Expo plugin configuration `app.config.js` ([#4583](https://github.com/getsentry/sentry-react-native/pull/4583))
```

Comment:
Probably this is too extended and can be included just in [the documentation](https://docs.sentry.io/platforms/react-native/manual-setup/expo/gradle/)